cluster/gce/gci/configure-helper.sh

```diff
@@ -2312,7 +2329,7 @@ function start-kube-addons {
     if [ -n "${CUSTOM_KUBE_PROXY_YAML:-}" ]; then
       # Replace with custom GKE kube proxy.
       cat > "$src_dir/kube-proxy/kube-proxy-ds.yaml" <<EOF
-$(echo "$CUSTOM_KUBE_PROXY_YAML")
+$CUSTOM_KUBE_PROXY_YAML
```

Comment:
this looks to be unrelated to calico

Reply:
Thanks for pointing it out! Reverted. PTAL